apps/boilerplate-front-end/src/components/facets/components/CurrentRefinement.jsx

```diff
@@ -122,3 +96,42 @@ function CurrentRefinementGeneral(props) {
 }
 
 export default CurrentRefinements
+
+// KEEPING THIS CODE JUST IN CASE
+//____________________________________________________________________
```

Comment:
Why?

Reply:
Will remove it after the reviews. I just want to make sure I didn't omit a case 😊